scripts/strip.sh

```diff
@@ -17,7 +17,8 @@ dynsyms="$orig.dynsyms"
 funcsyms="$orig.funcsyms"
 keep_symbols="$orig.keep_symbols"
 mini_debuginfo="$orig.minidebug"
-remove_sections=`readelf -W -S "$debuginfo" | awk '{ if (index($2,".debug_") != 1 && ($3 == "PROGBITS" || $3 == "NOTE" || $3 == "NOBITS") && index($8,"A") == 0) printf "--remove-section "$2" " }'`
+# silence "Error: Unable to find program interpreter name" warning from readelf, see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1012107
+remove_sections=`readelf -W -S "$debuginfo" 2> /dev/null | awk '{ if (index($2,".debug_") != 1 && ($3 == "PROGBITS" || $3 == "NOTE" || $3 == "NOBITS") && index($8,"A") == 0) printf "--remove-section "$2" " }'`
```

Comment:
It could cause problems if we have a real error and then don't see it, but I think it's unlikely and it's not worth spending effort to handle it.

Reply:
agreed. we could even filter out the error message , but not sure if it's worthwhile .